unity/Assets/Resources/Prefabs/NotGuarded.prefab

```diff
@@ -0,0 +1,92 @@
+%YAML 1.1
```

Comment:
I think this prefab is better located within the `ArtGallery` directory just like the other prefabs for the art gallery.

Reply:
I agree that it is better located there, but because the algorithm that checks for guard visibility is in a script that is not attached to a GameObject it thus is not accessible throughout the inspector and I can therefore not assign it to a public/serializedobject like we have with the lighthouse sprite. To my knowledge the only way of instantiating an object otherwise is using Resource.load, which requires whatever one wants to load to be within this Resource folder unfortunatly. 

Do you perhaps know of some other way of instantiating prefabs from a script not attached to a game object? Otherwise we would have to either move that algorithm to a script that is attached to a game object, or perhaps slightly change how it works: it currently returns a true of false based on whether all guards are gaurded, but we could also let it return a list with all unguarded guard's [locations] (to a script that is attachd to an in game object) and then plainly check there if the list has a length of 0 then all are guarded, otherwise spawn the prefabs on location like we have know.